pbiapi/pbiapi.py

```diff
@@ -104,20 +104,38 @@ def create_workspace(self, name):
             self.force_raise_http_error(response)
 
     @check_token
-    def add_users_to_workspace(self, workspace_name, users):
+    def add_user_to_workspace(self, workspace_name, user):
         self.get_workspaces()
         workspace_id = self.find_workspace_id_by_name(workspace_name, raise_if_missing=True)
 
-        # Workspace exists, lets add users:
+        # Workspace exists, lets add user:
         url = self.base_url + f"groups/{workspace_id}/users"
-        response = requests.post(url, data=users, headers=self.headers)
+        response = requests.post(url, data=user, headers=self.headers)
 
         if response.status_code == HTTP_OK_CODE:
             print(f"Added users to workspace '{workspace_name}'")
         else:
             print(f"Failed to add users to workspace '{workspace_name}':")
             self.force_raise_http_error(response)
 
+    @check_token
+    def get_users_from_workspace(self, name):
+        self.get_workspaces()
+        workspace_id = self.find_workspace_id_by_name(name)
+
+        if workspace_id:
+            url = "https://api.powerbi.com/v1.0/myorg/groups/{groupId}/users".format(groupId=workspace_id)
```

Comment:
I have nothing against using the `.format` syntax, but as we have changed to using `fstrings` everywhere else, you might as well change this one too